scripts/db_migrator.py

```diff
@@ -1090,7 +1109,8 @@ def version_4_0_4(self):
         Version 4_0_4.
         """
         log.log_info('Handling version_4_0_4')
-
+        # Updating GNMI table
+        self.migrate_gnmi()
```

Comment:
Adding to an existing version is not safe. What if this operation was executed on a device running an older sonic image which already has database version reached 4.0.4?

Reply:
Fixed